pkg/k8shandler/configuration_tmpl.go

```diff
@@ -15,7 +15,7 @@ node:
   max_local_storage_nodes: 1
 
 network:
-  host: 0.0.0.0
+  host: _global_
```

Comment:
looking at the es docs:
```
 network.host

    The node will bind to this hostname or IP address and publish (advertise) this 
    host to other nodes in the cluster. Accepts an IP address, hostname, a special value, 
    or an array of any combination of these. Note that any values containing a : 
    (e.g., an IPv6 address or containing one of the special values) must be quoted 
    because : is a special character in YAML. 0.0.0.0 is an acceptable IP address and 
    will bind to all network interfaces. The value 0 has the same effect as the value 0.0.0.0.

    Defaults to _local_.
```

Reply:
I'm not really certain what the difference is between `_site_` and `_global`.  I'm guessing maybe the address class to which it tries to bind?  In our case does it really matter given we are in a containerized environment where the platform is setting up the network for us, limits the addresses we get.  Using the `0.0.0.0` notation represents "bind to any interface" right?  Are we not effectively getting the same thing by allowing ES boot process to choose our binding?

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/6.8/modules-network.html#network-interface-values

i think we would want to use `["0.0.0.0", "::"]`